firmware_configurations/klipper/RealDeuce/MKS-Makerbase/Monster8_v1.0_003/Voron2_Monster8_Config.cfg

```diff
@@ -0,0 +1,531 @@
+#   Homing and Gantry Adjustment Routines
+#####################################################################
+
+[idle_timeout]
+timeout: 1800
+
+[safe_z_home]
+## XY Location of the Z Endstop Switch
+## Update -10,-10 to the XY coordinates of your endstop pin
+## (such as 157,305) after going through Z Endstop Pin
+## Location Definition step.
+home_xy_position:-10,-10
+speed:100
+z_hop:10
+
+[quad_gantry_level]
+## Use QUAD_GANTRY_LEVEL to level a gantry.
+## Min & Max gantry corners - measure from nozzle at MIN (0,0) and
+## MAX (250, 250), (300,300), or (350,350) depending on your printer size
+## to respective belt positions
+
+#--------------------------------------------------------------------
+##      Gantry Corners for 250mm Build
+##      Uncomment for 250mm build
+#gantry_corners:
+#       -60,-10
+#       310, 320
+##      Probe points
+#points:
+#       50,25
+#       50,175
+#       200,175
+#       200,25
+
+##      Gantry Corners for 300mm Build
+##      Uncomment for 300mm build
+#gantry_corners:
+#       -60,-10
+#       360,370
+##      Probe points
+#points:
+#       50,25
+#       50,225
+#       250,225
+#       250,25
+
+##      Gantry Corners for 350mm Build
+##      Uncomment for 350mm build
+#gantry_corners:
+#       -60,-10
+#       410,420
+##      Probe points
+#points:
+#       50,25
+#       50,275
+#       300,275
+#       300,25
+
+#--------------------------------------------------------------------
+speed: 100
+horizontal_move_z: 10
+retries: 5
+retry_tolerance: 0.0075
+max_adjust: 10
+
+#####################################################################
+#   Displays
+#####################################################################
+
+#--------------------------------------------------------------------
+
+[display]
+#! In EXP1/EXP2 (mini12864 LCD Display)
+lcd_type: uc1701
+cs_pin: PE11
+a0_pin: PD10
+rst_pin: PD9
+encoder_pins: ^PE8,^PE9
+click_pin: ^!PE10
+contrast: 63
+#spi_bus: spi1
+spi_software_mosi_pin: PA7
+spi_software_miso_pin: PA6
+spi_software_sclk_pin: PA5
+
+[neopixel mks_mini12864_v3]
+# To control Neopixel RGB in mini12864 display
+pin: PD8
+chain_count: 3
+initial_RED: 0.1
+initial_GREEN: 0.5
+initial_BLUE: 0.0
+color_order: RGB
+
+# Set RGB values on boot up for each Neopixel.
+# Index 1 = display, Index 2 and 3 = Knob
+[delayed_gcode setdisplayneopixel]
+initial_duration: 1
+gcode:
+        SET_LED LED=mks_mini12864_v3 RED=1 GREEN=1 BLUE=1 INDEX=1 TRANSMIT=0
+        SET_LED LED=mks_mini12864_v3 RED=1 GREEN=0 BLUE=0 INDEX=2 TRANSMIT=0
+        SET_LED LED=mks_mini12864_v3 RED=1 GREEN=0 BLUE=0 INDEX=3
+
+[output_pin beeper]
+pin: PB2
+
+#--------------------------------------------------------------------
+
+
+#####################################################################
+#   Macros
+#####################################################################
+
+[gcode_macro G32]
+gcode:
+    BED_MESH_CLEAR
+    G28
+    QUAD_GANTRY_LEVEL
+    G28
+    G0 X175 Y175 Z30 F3600
```

Comment:
add options for other printer sizes back in

Reply:
Whoops, good catch.